base/gfspapp/app_options.go

```diff
@@ -305,42 +305,42 @@ func DefaultGfBsDBOption(app *GfSpBaseApp, cfg *gfspconfig.GfSpConfig) error {
 			if val, ok := os.LookupEnv(bsdb.BsDBPasswd); ok {
 				cfg.BsDB.Passwd = val
 			}
-			if val, ok := os.LookupEnv(bsdb.BsDBAddress); ok {
-				cfg.BsDB.Address = val
-			}
-			if val, ok := os.LookupEnv(bsdb.BsDBDatabase); ok {
-				cfg.BsDB.Database = val
-			}
-			if val, ok := os.LookupEnv(bsdb.BsDBSwitchedUser); ok {
-				cfg.BsDBBackup.User = val
-			}
-			if val, ok := os.LookupEnv(bsdb.BsDBSwitchedPasswd); ok {
-				cfg.BsDBBackup.Passwd = val
-			}
-			if val, ok := os.LookupEnv(bsdb.BsDBSwitchedAddress); ok {
-				cfg.BsDBBackup.Address = val
-			}
-			if val, ok := os.LookupEnv(bsdb.BsDBSwitchedDatabase); ok {
-				cfg.BsDBBackup.Database = val
-			}
+			//if val, ok := os.LookupEnv(cfg.BlockSyncer.DBAddress); ok {
+			//	cfg.BsDB.Address = val
+			//}
+			//if val, ok := os.LookupEnv(cfg.BlockSyncer.DBName); ok {
+			//	cfg.BsDB.Database = val
+			//}
+			//if val, ok := os.LookupEnv(bsdb.BsDBSwitchedUser); ok {
+			//	cfg.BsDBBackup.User = val
+			//}
+			//if val, ok := os.LookupEnv(bsdb.BsDBSwitchedPasswd); ok {
+			//	cfg.BsDBBackup.Passwd = val
+			//}
+			//if val, ok := os.LookupEnv(bsdb.BsDBSwitchedAddress); ok {
+			//	cfg.BsDBBackup.Address = val
+			//}
+			//if val, ok := os.LookupEnv(bsdb.BsDBSwitchedDatabase); ok {
+			//	cfg.BsDBBackup.Database = val
+			//}
```

Comment:
If we are sure these are not needed, we shall remove the code other than comment them out.

Reply:
Delete the configuration first, leave the code, and add it back later depending on the situation

Reply:
+1
Const, Can you just remove them?
If needed, you can find them via git histories with searching commit messages.